x-pack/plugins/siem/public/components/page/network/users_table/index.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { isEqual } from 'lodash/fp';
+import React from 'react';
+import { connect } from 'react-redux';
+import { ActionCreator } from 'redux';
+import { StaticIndexPattern } from 'ui/index_patterns';
+
+import { FlowTarget, UsersEdges, UsersFields, UsersSortField } from '../../../../graphql/types';
+import { networkActions, networkModel, networkSelectors, State } from '../../../../store';
+import { Criteria, ItemsPerRow, LoadMoreTable, SortingBasicTable } from '../../../load_more_table';
+import { CountBadge } from '../../index';
+
+import { getUsersColumns } from './columns';
+import * as i18n from './translations';
+import { assertUnreachable } from '../../../../lib/helpers';
+
+interface OwnProps {
+  data: UsersEdges[];
+  flowTarget: FlowTarget;
+  loading: boolean;
+  hasNextPage: boolean;
+  indexPattern: StaticIndexPattern;
+  ip: string;
+  nextCursor: string;
+  totalCount: number;
+  loadMore: (cursor: string) => void;
+  type: networkModel.NetworkType;
+}
+
+interface UsersTableReduxProps {
+  usersSortField: UsersSortField;
+  limit: number;
+}
+
+interface UsersTableDispatchProps {
+  updateUsersLimit: ActionCreator<{
+    limit: number;
+    networkType: networkModel.NetworkType;
+  }>;
+  updateUsersSort: ActionCreator<{
+    usersSort: UsersSortField;
+    networkType: networkModel.NetworkType;
+  }>;
+}
+
+type UsersTableProps = OwnProps & UsersTableReduxProps & UsersTableDispatchProps;
+
+const rowItems: ItemsPerRow[] = [
+  {
+    text: i18n.ROWS_5,
+    numberOfRow: 5,
+  },
+  {
+    text: i18n.ROWS_10,
+    numberOfRow: 10,
+  },
+  {
+    text: i18n.ROWS_20,
+    numberOfRow: 20,
+  },
+  {
+    text: i18n.ROWS_50,
+    numberOfRow: 50,
+  },
+];
+
+export const UsersTableId = 'users-table';
```

Comment:
nit: I didn't use a capital letter for my table id. I know we do this in a few spots but it feels weird to use a captial letter for something non-JSX, non-FunctionalComponent.

Up to you if you want to change it.